components/TextFields/tests/unit/ContainedInputView/MDCBaseTextFieldTests.m

```diff
@@ -184,4 +184,67 @@ - (void)testClearButtonRectForBounds {
   XCTAssertTrue(CGRectEqualToRect(actualClearButtonFrame, expectedClearButtonFrame));
 }
 
+- (void)testFloatingLabelColorAccessors {
```

Comment:
Are we going to add some tests to make sure that setting the labels results in visual changes? This could be a snapshot test.

Reply:
Added a snapshot test for the floatingLabelColor accessor here: https://github.com/material-components/material-components-ios/pull/8406/files?short_path=bbaeb1f#diff-bbaeb1f9da4e514834f3ac6b04a1b234

Reply:
Cool we should add some for each state to make sure that setting the various colors for various states results in visual changes of the component.